src/main/xerus/monstercat/api/Player.kt

```diff
@@ -166,7 +170,12 @@ object Player: FadingHBox(true, targetHeight = 25) {
 				setOnReady {
 					label.text = "Now Playing: $track"
 					if(!Files.isDirectory(Settings.PLAYEREXPORTFILE())) {
-						Files.write(Settings.PLAYEREXPORTFILE(), arrayListOf("$track"), StandardOpenOption.CREATE)
+						Files.newBufferedWriter(Settings.PLAYEREXPORTFILE(),
+								StandardOpenOption.TRUNCATE_EXISTING,
+								StandardOpenOption.WRITE,
+								StandardOpenOption.CREATE
+						).close()
+						Files.write(Settings.PLAYEREXPORTFILE(), track.toString().toByteArray())
```

Comment:
Have you tried kotlin's inbuild file writing? Also, I think you can pass the `StandardOpenOption` directly to `Files.write`

Reply:
Doesn't work on Windows. Seems to be an bug within JVM. I've tried passing StandardOpenOption.TRUNCATE_EXISTING which should in theory rewrite the file completely but it simply writes over it, so writing smaller chunk simply writes one character over the old one.

Default options of Files.write should be TRUNCATE_EXISTING, WRITE, CREATE tho. Don't know why I forcefully put them in the bufferedwriter and not the write.

Reply:
Fixed in 9a4e5f5f81bf88c66e3fce8ea1b4ac5fd50152a2